_i18n/en.yml

```
@@ -515,7 +515,7 @@ faq:
   qavoidbc: Can I avoid downloading the entire blockchain?
   aavoidbc: "Yes. You don't need to download the @blockchain to transact on the network. You can connect to a @remote-node, which stores the blockchain for you. All the most common @wallets (including GUI and CLI) allow to use remote nodes to transact on the network. There are multiple ways to take advantage of this functionality. For example GUI and CLI offer a 'bootstrap node' feature, which allow people to download their own blockchain while using a remote node to immediately use the network. Ways to improve the usability of the Monero network are constantly being explored."
   qscanned: Why my wallet needs to be scanned everytime I open it?
-  ascanned: Because new @transactions have been recorded on the @blockchain from the last time you opened your wallet, which needs to scan all of them to make sure non of those transaction is yours. This process is not necessary in a mymonero-style (openmonero) wallet, a central server (which could be managed by you) does this work for you.
+  ascanned: Because new @transactions have been recorded on the @blockchain from the last time you opened your wallet, which needs to scan all of them to make sure non of those transaction is yours. This process is not necessary in a wallet software such as Edge wallet, a central server (which could be managed by you) does this work for you.
```

```suggestion
  ascanned: A new @block is added to the @blockchain approximately every 2 minutes. Your wallet will need to need to scan these blocks to find @transactions which belong to it. This process is not necessary when using a light wallet, also known as a remote synchronization wallet. For light wallets, a remote server (which could be managed by you) does the scanning on your behalf.
```

this is a bit better than the original text